specification/cognitiveservices/data-plane/AzureOpenAI/inference/2023-09-01-preview/generated.yaml

```diff
@@ -0,0 +1,1914 @@
+openapi: 3.0.0
```

Comment:
I just wanted to express some happiness about the readability of yaml 😄 and the OpenAPI version bump too!